packages/cloud_firestore/cloud_firestore_web/pubspec.yaml

```diff
@@ -11,7 +11,12 @@ environment:
 
 dependencies:
   _flutterfire_internals: ^1.3.7
-  cloud_firestore_platform_interface: ^5.16.2
```

Comment:
Are we truly using a platform that is a major version and **many** minor revisions old?

Reply:
This is now a "tag" and it comes from cloud_firestore version here:
packages/cloud_firestore/cloud_firestore/pubspec.yaml